contracts/BC_fusion/BSCGovernor.sol

```diff
@@ -107,40 +77,7 @@ contract BSCGovernor is
 
         // TODO
         // Different address will be set depending on the environment
-        governorProtector = address(0xdEaD);
-    }
-
-    /*----------------- onlyGovernorProtector -----------------*/
-    /**
-     * @dev Pause the whole system in emergency
-     */
-    function pause() external onlyGovernorProtector {
-        paused = true;
-        emit Paused();
-    }
-
-    /**
-     * @dev Resume the whole system
-     */
-    function resume() external onlyGovernorProtector {
-        paused = false;
-        emit Resumed();
-    }
-
-    /**
-     * @dev Add an address to the black list
-     */
-    function addToBlackList(address account) external onlyGovernorProtector {
-        blackList[account] = true;
-        emit BlackListed(account);
-    }
-
-    /**
-     * @dev Remove an address from the black list
-     */
-    function removeFromBlackList(address account) external onlyGovernorProtector {
-        blackList[account] = false;
-        emit UnBlackListed(account);
+        __Protectable_init_unchained(address(0xdEaD));
```

Comment:
Do you need change some script to sed the address here? I guess if any script, it is break now

Reply:
You're right. Fixed